frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -995,10 +987,7 @@ export const dashboardLogic = kea<dashboardLogicType>([
                 }
 
                 refreshesFinished += 1
-                if (refreshesFinished === insights.length) {
-                    breakpoint()
-                    cache.abortController = null
-
+                if (!cancelled && refreshesFinished === insights.length) {
```

Comment:
@macobo one last thing to check now I have the tests passing

I've had to add a `!cancelled` gate here because sometimes this conditional was executed after tests had finished running.

I think it's OK not to report time_to_see data for the dashboard load here since we'd be reporting on the cancellation but....

Reply:
Thanks for the heads up.